js/admin.js

```diff
@@ -104,6 +104,37 @@ var documentsSettings = {
 		);
 	},
 
+	saveWatermarkText: function(value) {
+		$.post(
+			OC.filePath('richdocuments', 'ajax', 'admin.php'),
+			{ 'watermark_text': value }
+		);
+
+		OC.Notification.showTemporary(t('richdocuments', 'Saved watermark'), {timeout: 2});
+	},
+
+	saveSecureViewOption: function(value) {
+		// FIXME: if version too low, throw error that this permission cannot be set for version <10.2 ?
```

Comment:
another alternative is setting min-version to 10.2 for the app, but 10.2 isn't out yet. we could merge this later

Reply:
or is there proper error handling on the UI in case this call failed ?

you can also add a version check, see in the `OC` namespace you might be able to find the current OC version...

Reply:
@PVince81 hmm - true will check compatibility with earlier version. yes, lets merge it after 10.2 is out.

Reply:
it is already done (min 10.2)